images/manageiq-base-worker/container-assets/entrypoint

```diff
@@ -11,6 +11,8 @@ KEY
 
 echo "${GUID}" > ${APP_ROOT}/GUID
 
-[[ -n $EMS_IDS ]] && WORKER_OPTIONS="--ems-id=${EMS_IDS} "
+WORKER_OPTIONS=""
+[[ -n $EMS_IDS ]] && WORKER_OPTIONS="${WORKER_OPTIONS} --ems-id=${EMS_IDS} "
+[[ -n $HOSTNAME ]] && WORKER_OPTIONS="${WORKER_OPTIONS} --system-uid=$HOSTNAME "
```

Comment:
This might affect docker monolithic, so we should just verify.

Reply:
Do we still support monolithic on master/k?  I have no idea how to build or test it.

Reply:
Yes, well, we "support" it 😄 .  It's more meant for "trying" out ManageIQ.

Reply:
if you can build the pods images, then it's just a regular Docker build from the core repo - it depends on the manageiq-ui-worker image - https://github.com/ManageIQ/manageiq/blob/master/Dockerfile#L2

Reply:
I'll try to build it tomorrow.

Reply:
is it viable to drop support for the monolith?
would be nice if we could reduce the number of different packaging options that we have.

Reply:
No, the monolith is used by a lot of people, including myself for trying things out...it's the simplest route as it's just `docker run manageiq/manageiq`

Reply:
Worked with this with LJ, and the key insight is that the system_uid value is being set in the _entrypoint_, however the entrypoint is overwritten with a custom on in the docker monolithic image.

Everything looks good :)